includes/Modules/AdSense.php

```diff
@@ -862,4 +864,17 @@ public static function normalize_client_id( $account_id, $client_id ) {
 		return 'accounts/' . $account_id . '/adclients/' . $client_id;
 	}
 
+	/**
+	 * Outputs the Adsense for Platforms meta tags.
+	 *
+	 * @since n.e.x.t
+	 */
+	private function render_amp_platform_meta_tags() {
```

Comment:
@kuasha420, IB says that it should be named as `render_platform_meta_tags`. Is there a reason why this has been called "render_**amp**_platform_meta_tags"?

Reply:
That's a mistake as far as I can tell. Will fix it in the next commit.